src/app/core/services/usage/firebase-analytics.service.ts

```diff
@@ -93,7 +93,7 @@ export class FirebaseAnalyticsService extends AnalyticsService {
               `Firebase User Property name ${key} is too long, cropping`
             ),
             this.crop(
-              String(value),
+              LogService.formatObject(value),
```

Comment:
Added this change because the attributes in Firebase were set as `[Object object]` instead of the value.